### DISCUSSION TITLE
(use case): Merge two similar projects into one

### DISCUSSION THREAD

**Discussion by ghost** [anuraghazra/github-readme-stats#2011] [Q&A] [answered]
Hi everyone. 

I would like to know if anyone thought of joining two projects: [github-profile-readme-generator](https://github.com/rahuldkjain/github-profile-readme-generator) ,  [github-readme-stats](https://github.com/anuraghazra/github-readme-stats) for a common goal: generating github profile files with the statistics. 

*(this is not a criticism but a comment from a user who likes both open source projects: github-profile-readme-generator and github-readme-stats :D)*

#### concept
![image](https://user-images.githubusercontent.com/111701513/189459169-97f30c9d-4906-4b58-b2e3-f625d4e59522.png)

In theory we can generate read-me with public-private statistics and other information as well. What do you all think about this idea?

**Comment by rickstaa** [comment, marked as answer]
@codehangen Thanks for your suggestions and for being active in the GRS community 🚀. I, however, don't think merging the two projects into one repository is wise from a maintainability perspective. Both projects serve different goals and have multiple code files. We can, however, merge them under the same organization if #1867 is implemented. However, this depends on whether @anuraghazra and @rahuldkjain are open to this idea.

**Reply by ghost** [reply]
Hi rickstaa, thank you for feedback.
